lib/rules/no-buffer-constructor.js

```diff
@@ -12,9 +12,25 @@
 /** @type {import('../shared/types').Rule} */
 module.exports = {
     meta: {
-        deprecated: true,
-
-        replacedBy: [],
+        deprecated: {
+            message: "Node.js rules were moved out of ESLint core.",
+            url: "https://eslint.org/docs/latest/use/migrating-to-7.0.0#deprecate-node-rules",
+            deprecatedSince: "7.0.0",
+            availableUntil: null,
+            replacedBy: [
+                {
+                    message: "eslint-plugin-n now maintains deprecated Node.js-related rules.",
+                    plugin: {
+                        name: "eslint-plugin-n",
+                        url: "https://github.com/eslint-community/eslint-plugin-n"
+                    },
+                    rule: {
+                        name: "no-buffer-constructor",
+                        url: "https://github.com/eslint-community/eslint-plugin-n/tree/master/docs/rules/no-buffer-constructor.md"
```

Comment:
```suggestion
                        name: "no-deprecated-api",
                        url: "https://github.com/eslint-community/eslint-plugin-n/tree/master/docs/rules/no-deprecated-api.md"
```

Per https://github.com/mysticatea/eslint-plugin-node/pull/206#discussion_r377549508